### DISCUSSION TITLE
How to make civetweb run on a specified network card

### DISCUSSION THREAD

**Discussion by maweicoltsmart** [zephyrproject-rtos/zephyr#44114] [General]
I have two network cards on my board, one is ethernet and the other is GSM Modem PPP. When I run civetweb, the startup process of GSM Modem PPP is interrupted and hangs for a long time. So I figured this problem would be solved if I made civetweb run on ethernet only.

**Comment by carlescufi** [comment]
@Nukersson and @rlubos FYI

**Comment by KozhinovAlexander** [comment]
Hi. Thank you for pointing to the problem. Try to look at civetweb's interface to communicate with network stack. I can imagine, it uses some older stuff since civetweb was not updated by me for a long time. 
Unfortunately, I can't do here more due to very taff schedule in other projects.

**Comment by ycsin** [comment]
> the startup process of GSM Modem PPP is interrupted and hangs for a long time

Do you mind elaborating this?

**Comment by maweicoltsmart** [comment]
Sorry, this is my own mistake, although civetweb doesn't run on the specified network card, it doesn't have the problem I mentioned earlier.
